### PR TITLE
Replace editor selectors

### LINKED ISSUE #1927: Unsupported markups no longer shown as disabled (gray)
Screenshot 2022-12-27 at 11 41 03

It used to be the case that markups that were not installed and thus not supported by a gollum instance would be shown in gray in the `select` element. This apparently no longer works (on master, so not related to #1915).

### PR DESCRIPTION
**Pull request by bartkamphorst**
This PR resolves #1927 by replacing the keybinding and format selectors by Primer SelectMenus. These menus can mirror select functionality but can be styled. See the screenshots below for the new look:

Screenshot 2023-08-31 at 13 55 36

Screenshot 2023-08-31 at 13 55 56

Screenshot 2023-08-31 at 13 56 07

### CODE REVIEW

**Comment by dometto** [comment]
This looks great!

**Review by dometto — APPROVED** [review]
I have no comments other than that the Capybara test should be fixed, but I don't immediately see how that should be done. :)

File: Rakefile [hunk 1]
```diff
@@ -74,8 +74,9 @@ def bundle_katex_fonts(assets_path)
   fonts_source = File.join(NODE_MODULES, fonts_subpath)
   fonts_target = File.join(assets_path, fonts_subpath)
   puts "\n  Copying KaTeX fonts from #{fonts_source} to #{fonts_target}..."
-  `mkdir #{fonts_target}`
-  `cp #{fonts_source}/*.woff2 #{fonts_target}/`
+  `mkdir "#{fonts_target}"`
```
Comment (dometto) [inline Rakefile:77]: Thanks for fixing this!

File: lib/gollum/templates/editor.mustache [hunk 2]
```diff
@@ -63,33 +64,76 @@
...
+        <details class="details-reset details-overlay">
+          <summary class="btn btn-sm btn-block" aria-haspopup="true">
+            {{#octicon}}pencil{{/octicon}}
+            <span class="mr-1">{{format}}</span>
+            <div class="dropdown-caret"></div>
+          </summary>
+          <div id="wiki_format" class="SelectMenu BasicSelect">
+            <div class="SelectMenu-modal">
+              <header class="SelectMenu-header">
+                <h3 class="SelectMenu-title text-left">Set this page's formatting to</h3>
```
Comment (dometto) [inline lib/gollum/templates/editor.mustache:109]: Actually, one thought: should this be `format` rather than `formatting`?

Comment (bartkamphorst) [reply lib/gollum/templates/editor.mustache]: Done.

**Comment by bartkamphorst** [comment]
@benjaminwil Any further comments/suggestions on this PR?

**Comment by benjaminwil** [comment]
I do not! Looks good. 😃

**Review by benjaminwil — APPROVED** [review]

**Pull request merged**